test/cmd/node-management.sh

```diff
@@ -35,10 +35,12 @@ run_cluster_management_tests() {
   "metadata": {
     "name": "test-pod-1",
     "labels": {
-      "e": "f"
+      "e": "f",
+      "type": "test-pod"
     }
   },
   "spec": {
+  "nodeName": "127.0.0.1",
```

Comment:
The `drain` tests in this file are fundamentally broken because of the lack of this label: they are draining a node with no pods assigned to it, which of course succeeds, but rather meaninglessly.